test/e2e/restart.go

```diff
@@ -216,6 +216,8 @@ func restartNodes(provider string, nt time.Duration) error {
 	switch provider {
 	case "gce":
 		return migRollingUpdateSelf(nt)
+	case "gke":
```

Comment:
maybe just fold this into the previous case statement with
`case "gce", "gke":`
?